lib/cog/command/pipeline/executor.ex

```diff
@@ -280,7 +283,8 @@ defmodule Cog.Command.Pipeline.Executor do
                                        # there's nothing to collect
                                        state.output
                                      body ->
-                                       state.output ++ Enum.map(List.wrap(body), &({&1, resp.template}))
+                                       state.output ++ Enum.map(List.wrap(body),
+                                                                &store_with_template(&1, resp.template))
```

Comment:
It's simple, but the same logic is used elsewhere, too; I wanted to encapsulate that functionality rather than have it in multiple places in not-so-greppable form.